lesson-3/3lesson_dz.py

```diff
@@ -0,0 +1,33 @@
+# 1. Використовуючи функцію print, надрукувати фразу “I love Python” 42 рази
```

Comment:
This file shouldn't be in this PR. Remove it, please, after you merge your previous PR

Reply:
files removed